pkg/cmd/release/create/create.go

```diff
@@ -820,7 +820,7 @@ func AskQuestions(octopus *octopusApiClient.Client, stdout io.Writer, asker ques
 	// we always need the deployment process, so we can prompt for package version overrides (or know that there aren't any packages, so it doesn't matter)
 
 	var gitReferenceKey string
-	if selectedProject.PersistenceSettings.GetType() == "VersionControlled" {
+	if selectedProject.PersistenceSettings.Type() == "VersionControlled" {
```

Comment:
there is a public const defined in the Go client for this, rather than comparing to a magic string

Reply:
yeah, I removed an unused one, will update to use the shared one

Reply:
fixed